apps/full-site-editing/full-site-editing-plugin/full-site-editing/blocks/site-description/edit.js

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable wpcalypso/jsx-classname-namespace */
+/**
+ * External dependencies
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { PlainText } from '@wordpress/editor';
+import { Component } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import apiFetch from '@wordpress/api-fetch';
+
+/**
+ * Internal dependencies
+ */
+
+class SiteDescriptionEdit extends Component {
+	state = {
+		fromApi: null,
+	};
+
+	componentDidMount() {
+		const { setAttributes } = this.props;
+		try {
```

Comment:
This is equivalent to `apiFetch.then( successHandler, rejectHandler )` or `apiFetch.then( successHandler ).catch( anyThrownErrorsFromErrorOrSuccessHandler );`